### PR TITLE
bitops non-default provider check

### PR DESCRIPTION
**Pull request by shyamrayaprolu**

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -87,7 +87,11 @@ echo "$PATH" >> ~/.bashrc
 
 # Setup cloud provider profile
 # TODO: check which cloudprovider
-/bin/bash $SCRIPTS_DIR/aws/setup.sh
+if [ -n "$PROVIDER_FLAG" ]; then
```
Comment (ConnorGraham) [inline scripts/deploy.sh:90]: What do you think of `NO_PROVIDER=true` or `SKIP_PROVIDER` or `PROVIDERS="none"` instead of  `PROVIDER_FLAG`?

Comment (shyamrayaprolu) [reply scripts/deploy.sh:90]: SKIP_PROVIDER or PROVIDERS="none" looks good.

Comment (dylan-bitovi) [reply scripts/deploy.sh:90]: +1 for SKIP_PROVIDER or PROVIDERS="none"

Comment (shyamrayaprolu) [reply scripts/deploy.sh]: ok changed the code to below 

```
if [ "$PROVIDERS" == "none" ]; then
  echo "running bitops for non-default provider..."
else
  /bin/bash $SCRIPTS_DIR/aws/setup.sh
fi
```

File: scripts/deploy.sh [hunk 2]
```diff
@@ -87,7 +87,17 @@ echo "$PATH" >> ~/.bashrc
...
-/bin/bash $SCRIPTS_DIR/aws/setup.sh
+# Default provider
+if [-z "$PROVIDERS"]; then
+    PROVIDERS="aws"
+fi
+
+# Provider switch statement
+if [ "$PROVIDERS" == "none" ]; then
+  echo "skipping providers and going straight to deployment tools"
+if [ "$PROVIDERS" == "aws" ]
```
Comment (ConnorGraham) [inline scripts/deploy.sh:98]: I don't think this is syntactically correct

**Review by ConnorGraham — CHANGES_REQUESTED** [review]

**Review by ConnorGraham — APPROVED** [review]

**Pull request merged**